gitbook/en/fallback.md

```diff
@@ -33,3 +47,43 @@ The following will be the output:
 ```html
 <p>hello world</p>
 ```
+
+
+## Explicit fallback with an array of locales
+
+It is possible to set more than one fallback locale by using an array of locales. For example
+
+```javascript
+fallbackLocale: [ 'fr', en' ],
+```
+
+
+## Explicit fallback with decision maps
+
+If more complex decision maps for fallback locales are required, it is possible to define decision maps with according fallback locales.
+
+Using the following decision map
+
+```javascript
+fallbackLocale: {
+  /* 1 */ 'de-CH':   ['fr', 'it'],
+  /* 2 */ 'zh-Hant': ['zh-Hans'],
+  /* 3 */ 'es-CL':   ['es-AR'],
+  /* 4 */ 'es':      ['en-GB'],
+  /* 5 */ 'pt':      ['es-AR'],
+  /* 6 */ 'default': ['en', 'da']
+},
+```
+
+will result in the following fallback chains
+
+| locale | fallback chains |
+|--------|-----------------|
+| `'de-CH'`   | de-CH > fr > it > en > da |
+| `'de'`      | de > en > da |
+| `'zh-Hant'` | zh-Hant > zh-Hans > zh > en > da |
```

Comment:
@mmokross 
In my understanding, I thin that not `zh-Hant > zh-Hans > zh > en > da`, but it should ``zh-Hant > zh-Hans > en > da`.
Right?

Reply:
sorry, I misunderstood.